ivy/functional/frontends/paddle/fft.py

```diff
@@ -100,3 +100,6 @@ def ifftshift(x, axes=None, name=None):
     roll = ivy.roll(x, shifts, axis=axes)
 
     return roll
+@to_ivy_arrays_and_back
+def ifft2(x, n=None, axis=(-2, -1), norm="backward", name=None):
+    return ifft2(x, n, axis, norm)
```

Comment:
There is no call from Ivy this seems to be a recursive call of the same function🙄 also this function isn't implemented on the Ivy API, yet you can use [`ifftn`](https://unify.ai/docs/ivy/docs/functional/ivy/experimental/layers/ivy.functional.ivy.experimental.layers.ifftn.html#ifftn) to implement this function